packages/unmock-core/src/service/dsl/dsl.ts

```diff
@@ -59,12 +58,11 @@ export abstract class DSL {
     }
     // Handles top-level schema and injects the literals to responses.
     // $code is a special case, handled outside this function (acts as a key and not a value)
-    if (top.$times !== undefined) {
-      throwOnErrorIfStrict(() => {
-        const translated = translate$times(top.$times);
-        injectUnmockProperty(responses, translated);
-      });
-    }
+    Object.keys(topTranslators)
```

Comment:
Great! One common pattern (that I am guilty of being too lazy to do) is the following:

```ts 
    Object.keys(topTranslators) 
      .filter(dslKey => top[dslKey] !== undefined /* no undefined tops */) 
      .map(dslKey => topTranslators[dslKey](top[dslKey]) /*gt the translation */) 
      .filter(translation => translation !== undefined /* no undefined translation */) 
```

etc...